api/v1alpha/client_example.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build ignore
+
+package main
+
+import (
+	"fmt"
+
+	// Note that if your project uses Godep to manage dependencies, then
+	// you need to change following the import paths.
+	"github.com/coreos/rkt/Godeps/_workspace/src/golang.org/x/net/context"
+	"github.com/coreos/rkt/Godeps/_workspace/src/google.golang.org/grpc"
+	"github.com/coreos/rkt/api/v1alpha"
+)
+
+func main() {
+	conn, err := grpc.Dial("localhost:15441", grpc.WithInsecure())
+	if err != nil {
+		panic(err)
+	}
+	c := v1alpha.NewPublicAPIClient(conn)
+	defer conn.Close()
+
+	// List pods.
+	podResp, err := c.ListPods(context.Background(), &v1alpha.ListPodsRequest{
+		// Specify the request here, e.g. We want to be returned with
```

Comment:
Specify the request: Fetch and print only running pods and their details.

(I find "return[ed] misleading here.)